MANIFEST.in

```diff
@@ -13,4 +13,4 @@ recursive-exclude * __pycache__
 recursive-exclude * *.py[co]
 
 recursive-include docs *.rst conf.py Makefile make.bat *.jpg *.png *.gif
-recursive-include cumulusci *.xml *.jar *.xsl *.txt *.properties *.sh *.py *.yml *.report *.md *.robot *.json *.css *.html
+recursive-include cumulusci *.xml *.jar *.xsl *.txt *.properties *.sh *.py *.yml *.report *.md *.robot *.json *.css *.html .gitignore
```

Comment:
Have cci project init been tested in an egg installation to make sure this works?

Reply:
Yeah, the tests on CircleCI run using tox so that builds and installs an sdist. It failed until I added this which is why I realized it was necessary.